docsteady/vcd.py

```diff
@@ -553,14 +557,18 @@ def do_req_coverage(ves, ve_coverage):
     """
     Calculate the coverage level of a requirement
     based on the downstram verification elements.
+    :param myvreq: version requirement name
```

Comment:
Eventually these should be changed to numpydoc everywhere so we can integrate into our sphinx system.

Reply:
i guess we could make a ticket